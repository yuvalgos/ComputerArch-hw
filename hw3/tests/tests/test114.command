./dflow_calc tests/test114.dat tests/test114.in p102 d173 d70 d68 d135 p177 d52 d61 p142 d74 d18 p28 d1 p73 p75 d159 d53 d78 p64 p41 d108 d187 d151 d97 d112 p36 d43 p46 p56 p98 p153 p199 p180 p196 p132 d142 d51 p172 p63 d194 p4 p19 d21 d130 d35 p19 p99 p120 p77 d50 d194 d165 d154 p28 d73 p3 p129 p60 p113 p126 p16 p15 p106 p149 d167 d59 d198 d72 d60 d168 d181 p79 d139 d113 d178 p157 d125 p123 d182 d112 p72 p171 p64 d193 p120 d112 d150 p59 d152 p159 d84 p43 d138 p46 d22 d82 p197 d122 d191 p28 p160 p119 d116 p196 d55 p66 d193 p125 d168 p39 p90 p194 d68 p153 d144 d139 d124 d157 p107 p49 d53 p138 p149 d171 d127 p169 d13 p75 d146 p187 p171 p82 p71 p126 d127 d110 p174 d167 p71 p15 d89