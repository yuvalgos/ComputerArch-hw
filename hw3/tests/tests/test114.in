7 8 14 4
0 14 10 4
1 25 21 8
2 12 24 16
4 18 3 25
1 26 31 18
1 13 11 2
0 30 24 30
1 9 2 10
2 13 30 15
6 0 27 1
7 7 2 4
7 15 20 27
4 8 16 31
5 12 0 22
3 25 4 10
8 4 4 12
7 4 29 5
7 8 7 25
7 18 17 21
8 30 30 28
1 30 26 28
0 26 20 19
8 25 13 20
3 14 8 2
2 14 30 14
5 27 10 14
7 6 1 14
0 20 17 22
0 14 10 13
5 25 16 15
0 19 16 16
6 11 0 15
6 22 18 1
2 1 18 15
0 17 19 25
3 29 2 16
0 18 27 11
2 13 13 4
8 12 22 10
3 13 26 15
5 25 15 27
8 18 14 8
5 9 30 0
7 14 18 1
6 29 0 19
8 27 2 11
3 18 6 25
2 29 23 13
8 3 10 5
6 18 28 9
8 12 15 6
5 28 2 6
3 10 8 29
8 15 8 2
1 4 6 0
7 31 30 25
0 7 15 22
2 4 0 28
7 9 14 12
5 14 6 6
0 31 27 7
0 9 20 12
2 30 24 17
6 17 27 9
2 19 23 19
4 2 24 17
6 30 28 18
2 16 7 27
2 3 13 12
5 31 23 14
7 5 3 15
5 29 27 22
4 28 20 10
1 31 19 23
5 26 27 17
6 10 9 20
6 6 18 17
2 5 29 22
8 5 17 0
8 29 22 0
8 12 2 25
5 14 2 8
4 27 3 15
8 1 22 21
4 30 8 30
6 3 13 10
8 11 15 11
2 25 6 30
3 18 29 6
0 16 24 11
0 19 19 25
5 12 6 5
0 20 5 25
2 12 2 5
8 10 0 30
1 12 17 2
2 5 21 17
3 30 12 25
1 1 5 9
6 9 30 16
6 14 29 17
1 18 24 8
7 7 29 2
5 8 21 10
0 30 17 19
5 26 12 30
4 24 0 24
2 13 19 27
4 26 16 18
1 23 4 27
5 15 22 24
3 17 30 18
0 6 3 15
1 13 16 12
0 9 30 22
4 25 14 5
5 17 1 20
0 2 31 3
3 7 21 6
6 2 19 24
7 13 24 31
1 16 18 23
1 12 31 6
0 4 3 12
1 30 4 11
7 13 5 27
1 10 12 6
1 29 6 22
2 15 24 4
0 31 10 7
6 12 28 5
8 10 12 1
2 23 26 24
8 13 26 21
8 30 18 9
4 8 17 26
1 7 14 2
5 19 30 28
5 0 27 5
5 21 30 24
3 3 15 24
1 21 16 8
7 11 8 30
8 7 12 25
5 7 25 0
6 4 27 1
3 24 25 6
0 16 29 22
2 7 1 25
6 14 8 1
0 7 8 25
4 19 21 30
3 27 10 11
0 0 1 3
5 24 27 23
2 22 20 16
0 2 8 10
4 12 21 14
2 1 27 29
2 7 15 12
7 18 25 30
1 20 30 0
7 19 21 28
5 23 25 3
7 19 12 23
4 12 29 6
3 21 14 0
7 5 7 19
7 18 9 6
4 21 29 9
1 4 4 20
4 29 23 9
5 0 30 17
3 8 10 7
8 16 2 1
3 11 14 19
0 24 14 20
6 10 19 16
6 27 7 8
1 17 30 15
4 4 6 30
2 24 0 15
8 17 25 26
6 19 24 26
7 27 28 12
1 7 20 30
5 17 6 14
0 2 18 7
8 12 22 18
1 20 17 23
0 31 15 18
8 23 25 31
2 1 12 17
8 12 26 11
8 11 9 23
7 19 20 3
6 9 3 6
2 1 28 27
5 5 31 9
