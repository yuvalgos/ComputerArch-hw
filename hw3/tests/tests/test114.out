Reading the opcodes latency info from tests/test114.dat ... Got latency for 9 opcodes
Reading the program file tests/test114.in ... Found 200 instructions
getProgDepth()==237
getDepDepth(102)==38
getInstDeps(173)=={135,117}
getInstDeps(70)=={-1,60}
getInstDeps(68)=={57,46}
getInstDeps(135)=={102,115}
getDepDepth(177)==177
getInstDeps(52)=={-1,27}
getInstDeps(61)=={46,57}
getDepDepth(142)==160
getInstDeps(74)=={65,-1}
getInstDeps(18)=={11,15}
getDepDepth(28)==0
getInstDeps(1)=={-1,-1}
getDepDepth(73)==45
getDepDepth(75)==111
getInstDeps(159)=={153,128}
getInstDeps(53)=={18,48}
getInstDeps(78)=={72,33}
getDepDepth(64)==94
getDepDepth(41)==63
getInstDeps(108)=={91,83}
getInstDeps(187)=={113,150}
getInstDeps(151)=={136,116}
getInstDeps(97)=={-1,64}
getInstDeps(112)=={105,102}
getDepDepth(36)==0
getInstDeps(43)=={21,10}
getDepDepth(46)==36
getDepDepth(56)==76
getDepDepth(98)==133
getDepDepth(153)==181
getDepDepth(199)==164
getDepDepth(180)==162
getDepDepth(196)==193
getDepDepth(132)==133
getInstDeps(142)=={122,136}
getInstDeps(51)=={12,27}
getDepDepth(172)==160
getDepDepth(63)==34
getInstDeps(194)=={109,176}
getDepDepth(4)==22
getDepDepth(19)==0
getInstDeps(21)=={5,-1}
getInstDeps(130)=={127,119}
getInstDeps(35)=={31,30}
getDepDepth(19)==0
getDepDepth(99)==116
getDepDepth(120)==135
getDepDepth(77)==111
getInstDeps(50)=={-1,43}
getInstDeps(194)=={109,176}
getInstDeps(165)=={158,164}
getInstDeps(154)=={99,141}
getDepDepth(28)==0
getInstDeps(73)=={28,53}
getDepDepth(3)==0
getDepDepth(129)==135
getDepDepth(60)==69
getDepDepth(113)==128
getDepDepth(126)==116
getDepDepth(16)==30
getDepDepth(15)==0
getDepDepth(106)==137
getDepDepth(149)==139
getInstDeps(167)=={150,154}
getInstDeps(59)=={44,51}
getInstDeps(198)=={73,185}
getInstDeps(72)=={46,33}
getInstDeps(60)=={27,27}
getInstDeps(168)=={160,165}
getInstDeps(181)=={113,135}
getDepDepth(79)==111
getInstDeps(139)=={83,97}
getInstDeps(113)=={86,111}
getInstDeps(178)=={165,175}
getDepDepth(157)==160
getInstDeps(125)=={124,87}
getDepDepth(123)==130
getInstDeps(182)=={173,129}
getInstDeps(112)=={105,102}
getDepDepth(72)==59
getDepDepth(171)==165
getDepDepth(64)==94
getInstDeps(193)=={189,187}
getDepDepth(120)==135
getInstDeps(112)=={105,102}
getInstDeps(150)=={136,99}
getDepDepth(59)==92
getInstDeps(152)=={142,135}
getDepDepth(159)==188
getInstDeps(84)=={33,-1}
getDepDepth(43)==39
getInstDeps(138)=={135,73}
getDepDepth(46)==36
getInstDeps(22)=={-1,-1}
getInstDeps(82)=={66,18}
getDepDepth(197)==147
getInstDeps(122)=={102,110}
getInstDeps(191)=={129,169}
getDepDepth(28)==0
getDepDepth(160)==188
getDepDepth(119)==130
getInstDeps(116)=={101,97}
getDepDepth(196)==193
getInstDeps(55)=={27,10}
getDepDepth(66)==111
getInstDeps(193)=={189,187}
getDepDepth(125)==135
getInstDeps(168)=={160,165}
getDepDepth(39)==36
getDepDepth(90)==84
getDepDepth(194)==214
getInstDeps(68)=={57,46}
getDepDepth(153)==181
getInstDeps(144)=={131,116}
getInstDeps(139)=={83,97}
getInstDeps(124)=={86,123}
getInstDeps(157)=={136,132}
getDepDepth(107)==17
getDepDepth(49)==0
getInstDeps(53)=={18,48}
getDepDepth(138)==162
getDepDepth(149)==139
getInstDeps(171)=={146,162}
getInstDeps(127)=={123,113}
getDepDepth(169)==139
getInstDeps(13)=={-1,-1}
getDepDepth(75)==111
getInstDeps(146)=={83,99}
getDepDepth(187)==177
getDepDepth(171)==165
getDepDepth(82)==122
getDepDepth(71)==97
getDepDepth(126)==116
getInstDeps(127)=={123,113}
getInstDeps(110)=={58,83}
getDepDepth(174)==193
getInstDeps(167)=={150,154}
getDepDepth(71)==97
getDepDepth(15)==0
getInstDeps(89)=={80,77}
